packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -150,7 +147,7 @@ export function normalizeError(
     const { message, stack } = extractLogSafeErrorProperties(error, false /* sanitizeStack */);
     const fluidError: IFluidErrorBase = new SimpleFluidError({
         errorType: "genericError", // Match Container/Driver generic error type
-        fluidErrorCode: annotations.errorCodeIfNone ?? "none",
+        fluidErrorCode: "none",
```

Comment:
again. lets not add it

Reply:
Sure. This means we can't distinguish which errors have been "touched" via `normalize`, but I can't think of why we would care, the only thing that means now is that there may be other telemetry props added.

I'm going to do `""` to keep the field required for the sake of the type guard, since we can have old/new versions of this code and the type guard in play together.  If we want to make it actually `undefined` I'll stage that in subsequent changes.

Reply:
Actually the sneaky `untrustedOrigin` is still around to say an error went through `normalize`, not that I think we'll need it 🤷

Reply:
seems fine either way. you can also punt on it for now. if it becomes fluidCallSite, then allowing it to not exist makes sense, and only error we spawn should have it